scripts/fonts.json

```diff
@@ -0,0 +1,450 @@
+{
+  "font-families": {
+    "Gothic A1": ["regular", "700"],
+    "M PLUS Rounded 1c": ["regular", "700"],
+    "Noto Sans": ["regular", "700", "italic", "700italic"],
+    "Noto Sans Arabic": ["regular", "700"],
+    "Noto Sans Armenian": ["regular", "700"],
+    "Noto Sans Bengali": ["regular", "700"],
+    "Noto Sans Devanagari": ["regular", "700"],
+    "Noto Sans Ethiopic": ["regular", "600"],
+    "Noto Sans Georgian": ["regular", "700"],
+    "Noto Sans Gurmukhi": ["regular", "700"],
+    "Noto Sans Hebrew": ["regular", "700"],
+    "Noto Sans Kannada": ["regular", "700"],
+    "Noto Sans Lao": ["regular", "700"],
+    "Noto Sans Malayalam": ["regular", "700"],
+    "Noto Sans Myanmar": ["regular", "700"],
+    "Noto Sans Tamil": ["regular", "700"],
+    "Noto Sans Thaana": ["regular", "700"],
+    "Noto Sans Tifinagh": ["regular"],
+    "Noto Sans Telugu": ["regular", "700"],
+    "Noto Sans Thai": ["regular", "700"]
+  },
+  "bundle-font-stacks": {
+    "Noto Sans HK": ["regular", "700"],
+    "Noto Sans JP": ["regular", "700"],
+    "Noto Sans KR": ["regular", "700"],
+    "Noto Sans SC": ["regular", "700"],
+    "Noto Sans TC": ["regular", "700"]
+  },
+  "glyph-ranges": {
+    "arabic": [
+      [1536, 1791],
+      [1872, 1919],
+      [2160, 2303],
+      [64336, 65023],
+      [65136, 65279],
+      [69216, 69247],
+      [69312, 69375],
+      [126064, 126143],
+      [126208, 126287],
+      [126464, 126719]
+    ],
+    "armenian": [[1328, 1423]],
+    "bengali": [[2432, 2559]],
+    "bopomofo": [
+      [12544, 12591],
+      [12704, 12735]
+    ],
+    "burmese": [[4096, 4255]],
+    "canadian_aboriginal": [
+      [5120, 5487],
+      [6928, 6975],
+      [71216, 71231]
+    ],
+    "cyrillic": [
+      [1024, 1327],
+      [7296, 7311],
+      [11744, 11775],
+      [42560, 42655],
+      [65070, 65071],
+      [122928, 123023]
+    ],
+    "devanagari": [[2304, 2431]],
+    "ethiopic": [
+      [4608, 5017],
+      [11648, 11743],
+      [43776, 43823]
+    ],
+    "greek": [
+      [880, 1023],
+      [7936, 8191],
+      [65856, 65999],
+      [119296, 119375]
+    ],
+    "georgian": [
+      [4256, 4351],
+      [11520, 11567]
+    ],
+    "gurmukhi": [[2560, 2687]],
+    "han": [
+      [11904, 12031],
+      [12032, 12245],
+      [12272, 12283],
+      [12284, 12287],
+      [12288, 12351]
+    ],
+    "hangul": [
+      [4352, 4607],
+      [12592, 12687],
+      [43360, 43391],
+      [55216, 55295]
+    ],
+    "hebrew": [
+      [1424, 1535],
+      [64285, 64331]
+    ],
+    "hiragana": [[12352, 12447]],
+    "kanbun": [[12688, 12703]],
+    "kannada": [[3200, 3327]],
+    "kanji": [[19968, 40959]],
+    "katakana": [
+      [12448, 12543],
+      [12784, 12799]
+    ],
```

Comment:
Can we add [Noto Sans Khmer](https://fonts.google.com/noto/specimen/Noto+Sans+Khmer) for Khmer, which is prominent in Southeast Asia?

[<img src="https://user-images.githubusercontent.com/1231218/225484702-10431404-277f-484f-ab0e-bee7745e99df.png" width="400" alt="Phnom Peng">](https://zelonewolf.github.io/openstreetmap-americana/#map=7/11.782/104.944&language=en)

Are there any other scripts of national languages that we should account for?

Reply:
Khmer script added in 8459f0a:

![image](https://user-images.githubusercontent.com/3254090/225489310-d2e50f92-abf0-42e6-b6a6-0aaac8a9469a.png)

Reply:
Canadian Aboriginal Syllabics, Gujarati, Odia (Oriya), Punjabi, Sinhala, Tibetan

Reply:
Can you offer some examples of test locations where those scripts are tagged in the main `name` tag in order to test?

Reply:
[ ] Canadian Aboriginal Syllabics
[ ] Gujarati
[ ] Odia
[ ] Punjabi (should be covered by existing Gurmukhi script)
[ ] Sinhala
[ ] Tibetan

Reply:
You're right about Punjabi; I overlooked Gurmukhi.

Gujarati and Odia are the predominant languages of the Indian states of Gujarat and Odisha, respectively. Being in India, `name=*` labels are in English, so these won't appear outside of `name:gu=*` and `name:or=*`, neither of which are present in OpenMapTiles.

I can remember Sinhala being used for `name=*` in the part of Sri Lanka where it's spoken (and Tamil in the Tamil-speaking area), but the local community seems to have shifted consensus to English recently.

* Canadian Aboriginal Syllabics: [Iqaluit, Nunavut :canada:](https://zelonewolf.github.io/openstreetmap-americana/#map=11.77/63.7458/-68.5242)
* Tibetan: [Thimphu :bhutan:](https://zelonewolf.github.io/openstreetmap-americana/#map=9.54/27.4657/89.6317/0/8)

Reply:
If they're not in the name tag and not an an OMT supported language, we'll never see it appear. Probably possible to do an overpass query to find examples. I can add them in for completeness of course but it'll be untested and this isn't the first time I've mangled unicode ranges.

Reply:
Tibetan added in 549918b:

![image](https://user-images.githubusercontent.com/3254090/225790310-aba90196-fe02-43ed-af08-b8d679e44b42.png)

Reply:
Sinhala added in 6abaf19:

![image](https://user-images.githubusercontent.com/3254090/225792747-3b63786e-867e-4a66-b976-7ea4d196427a.png)

Reply:
I am currently rendering a new planet with more languages which should allow us to freely test some of these scripts (and also make the map more international)

Reply:
Tibetan on top with the new tileserver update:
![image](https://user-images.githubusercontent.com/3254090/226209004-15835266-11d1-4a6d-94ba-ea10208043e9.png)

Reply:
Adding Oryia script in 2facda0:
[![image](https://user-images.githubusercontent.com/3254090/226210504-90872f23-2c61-42df-a54a-2c11085789b1.png)](http://localhost:1776/#map=6/24.605/82.398&language=or)

Reply:
Adding Gujarati in 131e4c5:
[![image](https://user-images.githubusercontent.com/3254090/226211049-c736d19e-528f-4d9e-9e98-8f78c1c15760.png)](http://localhost:1776/#map=5/40.86/-81.09&language=gu)

Reply:
Looks good. There are problems with ligatures and combining characters in all of the above screenshots, but that seems to be downstream of maplibre/maplibre-gl-js#50.

Reply:
Agreed - the solution that that will likely require a different font technology than PBF fontstacks.